src/with-props.js

```diff
@@ -69,23 +69,22 @@ export const withProps = (
     static _observedAttributes: Array<string>;
     static _props: Object;
 
-    _connected: boolean;
-    _constructed: boolean;
```

Comment:
This was to ensure the constructor wasn’t called twice. The polyfill seemed to do this awhile back. Removed because there’s noooo way we should havrle to test for this.